tempesta_fw/t/unit/test_http_parser.c

```diff
@@ -836,7 +836,7 @@ TEST(http_parser, casesense)
 TEST(http_parser, hdr_token_confusion)
 {
 	FOR_REQ("GET / HTTP/1.1\r\n"
-		"Accept: */*text/html\r\n"
+		"Accept: */html*\r\n"
```

Comment:
See my other comment, `*/html` is prohibited, the `text/html*` should be also prohibited, since mixing asterisks and tokens is not allowed.